python/py-gpstime/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gpstime
+version             0.3.2
+
+platforms           darwin
+supported_archs     noarch
+license             GPL-3
+maintainers         gmail.com:stefan.countryman
+
+python.rootname     gpstime
+
+description         GPS-aware datetime module with leap-second fetching
+long_description    This package provides a gpstime package, including a \
+                    gpstime subclass of the built-in datetime class with the \
+                    addition of GPS access and conversion methods.  Leap \
+                    second data is provided by the ietf_leap_seconds module \
+                    that helps automatically maintain a local copy of the \
+                    IETF leap second list: \
+                    https://www.ietf.org/timezones/data/leap-seconds.list \
+                    A command-line GPS data conversion utility that uses the \
+                    gpstime module is also included.  It is a rough \
+                    work-alike to tconvert.
+                    
+homepage            https://git.ligo.org/cds/${python.rootname}
+
+master_sites        https://git.ligo.org/cds/${python.rootname}/-/archive/${version}/
+distname            ${python.rootname}-${version}
+extract.suffix      .tar.gz
```

Comment:
this is the default, so you don't need to specify it

Reply:
Thank you! Removing. Will push new commit after fixing livecheck.